libs/ngx-charts-on-fhir/src/lib/range-selector/range-selector.component.spec.ts

```diff
@@ -57,30 +57,34 @@ describe('RangeSelectorComponent', () => {
   });
 
   it('should calculate proper 1 month ago date from max layer date', async () => {
+    jasmine.clock().mockDate(new Date('2023-04-13'));
```

Comment:
@semicolin  The mock date is set using the jasmine.clock().mockDate() method, it will only affect the current test case and will not affect any other test cases that run afterwards. Additionally, once the current test case is completed, the mock date will be removed, and the system time will return to its original state. 
also tried using `clock().install()/uninstall` but gets two errors
1.`Failed to find element matching one of the following queries:
        (MatButtonToggleHarness with host element matching selector: ".mat-button-toggle" satisfying the constraints: host matches selector "[id='1 y']")`
  2.`Error: Jasmine Clock was unable to install over custom global timer functions. Is the clock already installed?`

Reply:
got it. thanks for the explanation